src/ParsePromise.js

```diff
@@ -26,11 +26,15 @@ var isPromisesAPlusCompliant = true;
  * @constructor
  */
 export default class ParsePromise {
-  constructor() {
+  constructor(closure) {
```

Comment:
:+1: absolutely. I was calling it "closure" before I read the spec and determined it was deemed an "executor"